src/django/api/services/opensearch.py

```diff
@@ -0,0 +1,21 @@
+import logging
+from opensearchpy import OpenSearch
+
+host = 'opensearch-single-node'
+port = 9200
+
+logger = logging.getLogger(__name__)
+
+client = OpenSearch(
+    hosts=[{'host': host, 'port': port}],
+    use_ssl=False,
+    verify_certs=False
+)
+
+def search_production_locations(query_body):
```

Comment:
Search functionality should be in a separate module to divide responsibilities, as there can be other special operations like deletions, reindexing, etc. I recommend creating an `opensearch` [package](https://realpython.com/python-modules-packages/#python-packages)(regular folder `services/opensearch`) and including `opensearch.py` and `search.py` (create one) in this package.